WireUI/Package.swift

```diff
@@ -10,7 +10,8 @@ let package = Package(
     products: [
         .library(name: "WireDesign", targets: ["WireDesign"]),
         .library(name: "WireReusableUIComponents", targets: ["WireReusableUIComponents"]),
-        .library(name: "WireUIFoundation", targets: ["WireUIFoundation"])
+        .library(name: "WireUIFoundation", targets: ["WireUIFoundation"]),
+        .library(name: "WireUserProfile", targets: ["WireUserProfile"])
```

Comment:
Why is this a new target? Is it only UI?

Reply:
I combined all WireUI targets into a single WireUI test plan which is run by the CI.